### DISCUSSION TITLE
Monterey Guidance Revision 2

### DISCUSSION THREAD

**Discussion by robertgendler** [usnistgov/macos_security#124] [General]
Included in this release are updated guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, DISA-STIG, NIST 800-171, CNSSI-1253, CIS macOS Benchmarks Level 1 and 2, and CIS Critical Security Controls Version 8 baselines for macOS Monterey (12.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/monterey_rev2'>Monterey Guidance Revision 2</a>.</em>

**Comment by Rs4178** [comment]
> Included in this release are updated guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, DISA-STIG, NIST 800-171, CNSSI-1253, CIS macOS Benchmarks Level 1 and 2, and CIS Critical Security Controls Version 8 baselines for macOS Monterey (12.0).
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/monterey_rev2'>Monterey Guidance Revision 2</a>.</em>
